sympy/ntheory/tests/test_modular.py

```diff
@@ -1,4 +1,5 @@
-from sympy.ntheory.modular import crt, crt1, crt2, solve_congruence, crt_cartesian
+from sympy.ntheory.modular import crt, crt1, crt2, solve_congruence, \
+    crt_cartesian
```

Comment:
@smichr I just tested the code, I think you forgot to import ```cartes``` from ```utilities.iterables```. I will add it in the next commit after running the tests